pop_db.py

```
@@ -1,69 +1,106 @@
-    # Add admin user
-    admin_user = User(
-        username="admin",
+    # Ensure admin user exists (idempotent)
+    ensure_admin(
         email="admin@posthog.com",
+        username="admin",
         plan="Premium",
         is_admin=True,
-        is_adult=False
+        is_adult=False,
+        password="admin",
```

Ruff (0.12.2)

Possible hardcoded password assigned to argument: "password"

(S106)